RoaringBitmap/src/main/java/org/roaringbitmap/longlong/Roaring64Bitmap.java

```diff
@@ -649,7 +649,12 @@ public void removeLong(long x) {
       char low = LongUtils.lowPart(x);
       Container container = containerWithIdx.getContainer();
       Container freshContainer = container.remove(low);
-      highLowContainer.replaceContainer(containerWithIdx.getContainerIdx(), freshContainer);
+      if (freshContainer.isEmpty()) {
+        // Since this prefix doesn't have a leaf, remove it from the tree
```

Comment:
We remove it not due to it doesn't have a leaf(in fact it has one). Here we can say : "To save the memory, remove the empty container"